### PR TITLE
Added PR Template, Updated .gitignore

### LINKED ISSUE #8: Create a PR Template for develop branch
- [x] PR template for project maintainers to contribute. Should contain the following:
- Description of the issue/ feature
- Which issue it closes
- Screenshots of working implementation
- The pull request is made to the `develop` branch
- `black` gave 0 errors
- `flake` gave 0 errors
- Squashed commits into a single one

### PR DESCRIPTION
**Pull request by m0mosenpai**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -127,3 +127,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Audio files for testing
+audio/
```
Comment (gmelodie) [inline .gitignore:132]: Is this related?

**Review by gmelodie — COMMENTED** [review]
I don't think having a PR template is super important right now (they're more useful when you have to deal with a lot of PRs coming from people that don't know about your coding standards). On the other hand, it's good gh practice :)

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+-[] Bug Fix (non-breaking change that fixes an issue)
+-[] New feature (non-breaking change which adds functionality)
+-[] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+# Checklist
+
+-[] The pull request is for branch `develop`
+-[] `black` gave 0 errors
+-[] `flake` gave 0 errors
```
Comment (gmelodie) [inline .github/pull_request_template.md:19]: These could be [git hooks](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks)

File: .gitignore (hunk 1 above)
Comment (m0mosenpai) [reply .gitignore:132]: Oh not with the template. We just updated the .gitignore. I guess I could have committed this in a different PR mb.

File: .github/pull_request_template.md (hunk 2 above)
Comment (m0mosenpai) [reply .github/pull_request_template.md:19]: Yeah was about to do that only. Thought we could explicitly mention in the template anyways but I guess it's fine.
